src/test/java/io/xstefank/wildfly/bot/PRRuleTitleBodyCheckTest.java

```diff
@@ -32,14 +33,14 @@ public class PRRuleTitleBodyCheckTest {
     void testTitleBodyCheckForTitle() throws IOException {
         gitHubJson = GitHubJson.builder(VALID_PR_TEMPLATE_JSON).build();
         wildflyConfigFile = """
-            wildfly:
-              rules:
-                - id: "TitleBody"
-                  title: "NonTitle"
-                  body: "NonBody"
-                  titleBody: "WFLY"
-                  notify: [7125767235]
-            """;
+                wildfly:
+                  rules:
+                    - id: "TitleBody"
+                      title: "NonTitle"
+                      body: "NonBody"
+                      titleBody: "WFLY"
+                      notify: [7125767235]
+                """;
```

Comment:
for future, try to avoid this kind of changes. You will be showing in blame even if you are not changing anything here.